apps/site-app/src/containers/App/styles.module.less

```diff
@@ -1,47 +1,56 @@
 @import "~@thecointech/site-semantic-theme/variables.less";
 
-#mainContent{
-  padding: 10px;
-  padding-left: 66px;
-  margin: auto;
-  width: 62%;
-  padding-top: 70px;
-}
+#app {
+
+  display              : grid;
+  grid-template-rows   : 60px minmax(450px, auto) 32px;
+  grid-template-columns: 135px 171px minmax(auto, 837px) 173px 135px;
+  grid-template-areas  :
+    "lgap header  header header rgap"
+    "lgap sidebar main   widgets  rgap"
+    "lgap footer  footer footer rgap";
+
+  .menuPanel {
+    grid-area: sidebar;
+  }
+
+  .mainContent {
+    grid-area: main;
+  }
 
-.contentContainer{
-  max-width: 1300px;
-  margin: auto;
+  .widgets {
+    grid-area: widgets;
+  }
 }
 
-.pageMainInner {
-  min-height: 450px;
-  padding: 0em;
+#mainContent {
+  padding: 10px;
 }
 
-#headerDecoration{
-  border-bottom: 1px solid @theCoinPalettePaleGreen2Faded50;
-  width: 100%;
+#headerDecoration {
+  border-bottom   : 1px solid @theCoinPalettePaleGreen2Faded50;
+  width           : 100%;
```

Comment:
Those spaces are weird. Was that added by your vscode?

Reply:
yeah - I think I added an auto-formatter recently.  I really need to add a prettier (or similar) hook